### DISCUSSION TITLE
Active directory user integration

### DISCUSSION THREAD

**Discussion by MPR-Git** [OctoFarm/OctoFarm#593] [Feature Request]
Hello

your project sound awesome.
We are looking for a tool just like your's in a professional environment.

An Active Directory implementation and SSO would be very cool !

But we will try your solution in a few weeks on 5 prusa and if it's great, maybe 45 prusa with octoprint  ;-)

**Comment by NotExpectedYet** [comment]
Thanks for the kind words! Let us know if you need any help. 

I believe Authelia or something similar would work with a reverse proxy setup. You'd just have to make sure your OctoPrint instances are reversed as well currently due to some legacy code in the client. 

I just had a quick scout and there seems to be a node package already built we could utilise so that may not be impossible for AD integration. I would like to get the user system bolstered a bit more as it's currently quite limited but will keep it in mind for that.
